packages/client/hmi-client/src/components/model/petrinet/tera-initial-table-v2.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+	<!-- Stratified -->
+	<template v-if="isStratified">
+		<Accordion multiple>
+			<AccordionTab
+				v-for="[key, values] in getUnstratifiedInitials(
```

Comment:
`getUnstratifiedInitials` is not exactly compatible with `isStratifiedModel`. The former is acting on the AMR and latter the MMT version of the AMR.

Slight preference would be to move towards the latter option, as it gives a little more freedom if we want to maintain regnet and stockflow compatibilities, since the MMT version handles all 3 but getUnstratifiedInitials only for petrinet. What you may want to look at here is collapseInitials from the mira folder


https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/model-representation/mira/mira.ts#L151C14-L167